layouts/partials/version-banner.html

```diff
@@ -0,0 +1,20 @@
+{{ $color := "primary" }}
+<!-- Check the variable that indicates whether this is an archived doc set.
+  If yes, display a banner. -->
+  {{ if .Site.Params.deprecated }} {{/* Change the `deprecated` param to Docsy provided `archived_version` param once all use of this param is updated */}}
+  {{ $latest_version := .Site.Params.url_latest_version }}
+  {{ $current_version := .Site.Params.version }}
+  <div class="pageinfo pageinfo-{{ $color }}">
```

Comment:
I suggest:
```suggestion
  <div class="pageinfo pageinfo-{{ $color }} deprecation-notice">
```

Reply:
What would the `deprecation-notice` class do? I don't see any associated CSS for it.

Reply:
This is a good place to bring out something I was thinking. Maybe we should name all our classes/ids with a prefix like `k8s-`. Docsy does it with its `td-` prefix. This will make it simple to know if a particular class/id is a custom one or provided by upstream. So, we can use `k8s-deprecation-notice` here. What do you say?

Reply:
Let's take that conversation to Slack @SayakMukhopadhyay

Reply:
I have added a `k8s-deprecation-warning` class.